tests/libcxx/tests.all

```diff
@@ -5375,19 +6221,30 @@
 ../../3rdparty/libcxx/libcxx/test/std/utilities/variant/variant.variant/variant.ctor/move.pass.cpp
 ../../3rdparty/libcxx/libcxx/test/std/utilities/variant/variant.variant/variant.ctor/T.pass.cpp
 ../../3rdparty/libcxx/libcxx/test/std/utilities/variant/variant.variant/variant.dtor/dtor.pass.cpp
+../../3rdparty/libcxx/libcxx/test/std/utilities/variant/variant.variant/variant_empty.fail.cpp
 ../../3rdparty/libcxx/libcxx/test/std/utilities/variant/variant.variant/variant.mod/emplace_index_args.pass.cpp
 ../../3rdparty/libcxx/libcxx/test/std/utilities/variant/variant.variant/variant.mod/emplace_index_init_list_args.pass.cpp
 ../../3rdparty/libcxx/libcxx/test/std/utilities/variant/variant.variant/variant.mod/emplace_type_args.pass.cpp
 ../../3rdparty/libcxx/libcxx/test/std/utilities/variant/variant.variant/variant.mod/emplace_type_init_list_args.pass.cpp
+../../3rdparty/libcxx/libcxx/test/std/utilities/variant/variant.variant/variant_reference.fail.cpp
 ../../3rdparty/libcxx/libcxx/test/std/utilities/variant/variant.variant/variant.status/index.pass.cpp
 ../../3rdparty/libcxx/libcxx/test/std/utilities/variant/variant.variant/variant.status/valueless_by_exception.pass.cpp
 ../../3rdparty/libcxx/libcxx/test/std/utilities/variant/variant.variant/variant.swap/swap.pass.cpp
+../../3rdparty/libcxx/libcxx/test/std/utilities/variant/variant.variant/variant_void.fail.cpp
 ../../3rdparty/libcxx/libcxx/test/std/utilities/variant/variant.visit/visit.pass.cpp
+../../3rdparty/libcxx/libcxx/test/support/external_threads.cpp
 ../../3rdparty/libcxx/libcxx/test/support/nothing_to_do.pass.cpp
 ../../3rdparty/libcxx/libcxx/test/support/test.support/test_convertible_header.pass.cpp
 ../../3rdparty/libcxx/libcxx/test/support/test.support/test_demangle.pass.cpp
+../../3rdparty/libcxx/libcxx/test/support/test.support/test_macros_header_exceptions.fail.cpp
 ../../3rdparty/libcxx/libcxx/test/support/test.support/test_macros_header_exceptions.pass.cpp
+../../3rdparty/libcxx/libcxx/test/support/test.support/test_macros_header_rtti.fail.cpp
 ../../3rdparty/libcxx/libcxx/test/support/test.support/test_macros_header_rtti.pass.cpp
 ../../3rdparty/libcxx/libcxx/test/support/test.support/test_poisoned_hash_helper.pass.cpp
 ../../3rdparty/libcxx/libcxx/test/support/test.workarounds/c1xx_broken_is_trivially_copyable.pass.cpp
 ../../3rdparty/libcxx/libcxx/test/support/test.workarounds/c1xx_broken_za_ctor_check.pass.cpp
+../../3rdparty/libcxx/libcxx/utils/google-benchmark/cmake/gnu_posix_regex.cpp
```

Comment:
This last block also don't look like tests, we should drop these.

Reply:
Shall I remove the whole block ?
../../3rdparty/libcxx/libcxx/utils/google-benchmark/cmake/gnu_posix_regex.cpp
../../3rdparty/libcxx/libcxx/utils/google-benchmark/cmake/posix_regex.cpp
../../3rdparty/libcxx/libcxx/utils/google-benchmark/cmake/std_regex.cpp
../../3rdparty/libcxx/libcxx/utils/google-benchmark/cmake/steady_clock.cpp
../../3rdparty/libcxx/libcxx/utils/google-benchmark/cmake/thread_safety_attributes.cpp

Reply:
Yes … these aren't actually tests and there's not point having them in tests.all and tests.unsupported.

Reply:
Removed from tests.all , tests.unsupported and tests.supported (there was one google-benchmark there also)